WalletWasabi.Fluent/Views/Wallets/Transactions/Outputs/Columns/IsOwnIndicatorView.axaml

```diff
@@ -0,0 +1,20 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:outputs="clr-namespace:WalletWasabi.Fluent.ViewModels.Wallets.Transactions.Outputs"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:DataType="outputs:OutputsCoinListItem"
+             x:CompileBindings="True"
+             x:Class="WalletWasabi.Fluent.Views.Wallets.Transactions.Outputs.Columns.IsOwnIndicatorView">
+
+  <InvalidatingStackPanel Orientation="Horizontal" Margin="0 0 0 0" Spacing="2" HorizontalAlignment="Right" ToolTip.Tip="This output belongs to one of your opened wallets">
+    <Border BorderThickness="1"
+            HorizontalAlignment="Right"
+            VerticalAlignment="Center"
+            IsVisible="{Binding IsOwn}"
+            BorderBrush="{DynamicResource TextControlForeground}" Opacity="0.6" CornerRadius="4" Padding="3">
+      <TextBlock Classes="h10" Text="OWN"/>
```

Comment:
having the `OWN` text only at outputs might cause confusion.
It is expected to also have it at inputs, i.e. all _OWN_ coins imo

although I'm not exactly sure how it could be improved as the own inputs have anonscore

or maybe remove it in full idk, wdyt?

Reply:
Having it as inputs is useless: All inputs shown are OWN, otherwise I can't show it

Reply:
yes, from a noob user perspective might be confusing when some have OWN, while others coins don't have it while they're also OWN. that's what I meant

Reply:
Won't fix